src/components/20-molecules/list/index.react.d.ts

```diff
@@ -6,7 +6,6 @@ export interface AXAListProps {
   variant?: string;
```

Comment:
You could specify the values like "orderd | ....."

Reply:
good point!